core/src/executor.rs

```diff
@@ -185,6 +198,15 @@ impl Executor {
     ) -> Result<(), ValidationFail> {
         trace!("Running instruction validation");
 
+        if 0 != state_transaction.height() {
+            let Ok(authority_account) = state_transaction.world.account(authority) else {
```

Comment:
This could be a function to reduce code duplication

Reply:
IMO this should be verified before calling executor. Iroha should never call `validate_*` in the 1st place if authority doesn't exist or is inactive